SignInCheckIn/SignInCheckIn/Services/EventService.cs

```diff
@@ -80,6 +80,33 @@ public EventDto GetCurrentEventForSite(int siteId)
         //    return Mapper.Map<MpEventDto, EventDto>(currentEvents.First());
         //}
 
```

Comment:
It claimed to be obsolete down in the room service so should we be using it or a stored proc?